scripts/bundleSize/pageTypeBundleExtractor.js

```diff
@@ -1,13 +1,26 @@
-const modernBundleReport = require('../../reports/modern.webpackBundleReport.json');
-const legacyBundleReport = require('../../reports/legacy.webpackBundleReport.json');
+import { readFile } from 'fs/promises';
```

Comment:
With ESM you can longer import json directly, you must read the file and parse the json